src/main/java/pl/gov/coi/pomocua/ads/legalassistances/LegalAssistanceResource.java

```diff
@@ -0,0 +1,36 @@
+package pl.gov.coi.pomocua.ads.legalassistances;
+
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+import java.util.Optional;
+
+
+@RestController
+@RequestMapping(path = "/api/", produces = "application/json")
+public class LegalAssistanceResource {
+
+    private final LegalAssistanceRepository legalAssistanceRepository;
+
+    public LegalAssistanceResource(LegalAssistanceRepository legalAssistanceRepository){this.legalAssistanceRepository = legalAssistanceRepository;}
```

Comment:
Just a mention: in other resource classes, `@RequiredArgsConstructor` was used that allows to omit writing this constructor by ourselves.
Or maybe you're a strong opponent of Lombok, then ignore my comment

Reply:
That sounds good, it will simplify IMO :)